vm/vm.go

```diff
@@ -156,7 +157,7 @@ func (vm *VM) Initialize(
 	}
 	vm.metrics = metrics
 	vm.proposerMonitor = NewProposerMonitor(vm)
-	vm.networkManager = NewNetworkManager(vm, appSender)
+	vm.networkManager = network.NewNetworkManager(vm.snowCtx.Log, appSender)
```

Comment:
This stutters...we should change `NewNetworkManager` to `New/NewManager`

Reply:
Will also rename struct NetworkManager to Manager and struct NetworkHandler to Handler